src/components/Header/Header.tsx

```diff
@@ -1,31 +1,36 @@
-import React from "react";
+import React, { useState } from "react";
 import {
   chakra,
   Box,
-  Flex,
   HStack,
-  Spacer,
   useMultiStyleConfig,
+  Spacer,
   VStack,
 } from "@chakra-ui/react";
 
-import Button from "../Button/Button";
-import HorizontalRule from "../HorizontalRule/HorizontalRule";
-import Icon from "../Icons/Icon";
 import Link from "../Link/Link";
-import List from "../List/List";
 import Logo from "../Logo/Logo";
+import HorizontalRule from "../HorizontalRule/HorizontalRule";
 import Notification from "../Notification/Notification";
 import useWindowSize from "../../hooks/useWindowSize";
+import UpperNav from "./components/UpperNav";
+import LowerNav from "./components/LowerNav";
+import Mobile from "./components/Mobile";
 
 export const Header = chakra(() => {
-  const styles = useMultiStyleConfig("Header", {});
   const windowDimensions = useWindowSize();
   // These values are based on the breakpoints md and lg
   // from the NYPL theme object.
   const breakpointMedium = 600;
   const breakpointLarge = 960;
 
+  const styles = useMultiStyleConfig("Header", {
+    width: windowDimensions.width,
+    breakpointMedium,
```

Comment:
Instead of passing these arguments and doing the logic check in the theme file, make a boolean for the logic check (`with < breakpointMedium`) at this top level and pass that boolean down. Same idea for the `windowDimensions.width > breakpointLarge`.